coding-guides/using-audit-logs.md

```diff
@@ -6,8 +6,7 @@ This guide will explain everything about audit logs and how to use them.
 
 The first thing that you will need is a working Discord Bot. If you do not have one, please visit [Your First Bot](getting-started/your-basic-bot.md) to get started.
 
-Now, some things to take note of. The bot will need some permissions. The main permission the bot will need
-is 'VIEW_AUDIT_LOGs'. This permission allows the bot to view the audit logs and do whatever you want to, obeying the Discord ToS that is. 
+Now, some things to take note of. This guide is using discord.js@11.3.2 The bot will need some permissions. The main permission the bot will need is `'VIEW_AUDIT_LOGS'`. This permission allows the bot to view the audit logs and do whatever you want to, obeying the Discord ToS that is. 
```

Comment:
_This guide is using discord.js@11.3.2 **T**he bot_, that `T` needs to follow a period, `.` or be lower case `t`